lib/chef_zero/endpoints/organization_user_endpoint.rb

```diff
@@ -20,29 +20,7 @@ def delete(request)
         json_response(200, ChefData::DataNormalizer.normalize_user(user, request.rest_path[3], ['username'], server.options[:osc_compat]))
       end
 
-      def post(request)
```

Comment:
The current chef server only support `POST /organization/ORGNAME/users/`.  I don't think there is a version that supports `POST /organizations/ORGNAME/users/USER` but I haven't look extensively.